assets/templates/colors-component.html

```diff
@@ -69,7 +69,314 @@
         This will be replaced by the content of navigation.html
         <!-- /build -->
         <div class="dpsg-content-wrap">
-          <h3>Colores</h3>
+          <article class="dpsg-wrap-colors">
+            <h3>El color como lenguage</h3>
+            <p>
+              Es muy importante la
+              <strong>homogeneidad de nuestros mensajes</strong> para
+              <strong>no confundir a nuestros Usuarios.</strong>
+            </p>
+            <p>
+              Hay que entender que nosotros comunicamos, orientamos y guiamos
+              <strong
+                >no sólo a traves de los contenidos sino también de los
+                colores.</strong
+              ><br />
+              Si todos utilizamos las mismas formas de comunicarnos evitaremos
+              enviar mensajes confusos.
+            </p>
+            <p></p>
+            <div class="dpsg-content-colors">
+              <div class="dpsg-color-card dpsg-white">
+                <div class="dpsg-icon-color">Color</div>
+                <div class="dpsg-content-color">
+                  <h3>white</h3>
+                  <p class="dpsg-code">
+                    Color: #ffffff;
+                  </p>
+                  <p class="dpsg-code">
+                    .dp-white
```

Comment:
Each time that we add a new color, we should remember to update this page. 

Sadly, I think that it is very fragile, and it will very easy having inconsistencies in the future.